Algorithms/Stacks/maxdiff.java

```diff
@@ -0,0 +1,75 @@
+//This code finds the maximum absolute difference between 
+//nearest left and right smaller element of every element in an array
+import java.util.*;
+import java.io.*;
+class maxdiff
+{
+    public static void main(String args[])
+    {
+        Scanner s=new Scanner(System.in);
```

Comment:
Use proper spaces everywhere as mentioned in this [section](https://github.com/sahilbansal17/Competitive_Coding/wiki/Coding-Standards#spaces) in coding standards.